src/storage/src/hummock/sstable/builder.rs

```diff
@@ -223,8 +227,11 @@ impl<W: SstableWriter> SstableBuilder<W> {
         self.total_key_count += 1;
         self.last_table_stats.total_key_count += 1;
 
-        self.block_builder
-            .add(self.raw_key.as_ref(), self.raw_value.as_ref());
+        self.block_builder.add(
+            self.raw_key.as_ref(),
+            self.raw_value.as_ref(),
+            self.not_only_visited_by_vnode_fetch,
```

Comment:
I prefer pass it when we create `block_builder`.

Reply:
> I prefer pass it when we create `block_builder`.

fixed